src/include/migraphx/op/pointwise.hpp

```diff
@@ -45,14 +45,19 @@ struct pointwise
         {
             MIGRAPHX_THROW("should have one submodule.");
         }
-        auto* pm    = mods.front();
+        auto* pm = mods.front();
+        if(pm->get_output_shapes().size() != 1)
+            MIGRAPHX_THROW("submodule should have only one output.");
+        // if pointwise op doens't have any input args (const-foldable) then just return output
```

Comment:
We shouldn't be creating a pointwise op in `fuse_pointwise`. Ideally, there should be a check for this.